src/modules.ts

```diff
@@ -8,6 +8,29 @@ interface Filters {
   whitelists: RegExp[];
 }
 
+/**
+ * Represents the `postFilter` function signature.
+ */
+export type PostFilterFunc = (
```

Comment:
For what it's worth, I chose **post**Filter because if there's ever a need to do pre-filtering, it won't conflict.